security/integration/main.go

```diff
@@ -317,7 +471,108 @@ func waitForSecretExist(secretName string, timeToWait time.Duration) (*v1.Secret
 				return secret, nil
 			}
 		case <-time.After(timeToWait - time.Since(startTime)):
-			return nil, fmt.Errorf("secret %v/%v did not become existent within %v", opts.namespace, secretName, timeToWait)
+			return nil, fmt.Errorf("secret %v/%v did not become existent within %v",
+				opts.namespace, secretName, timeToWait)
 		}
 	}
 }
+
+func readFile(path string) (string, error) {
```

Comment:
I think this main file is too big. Can you move some functions out to some new file?

Reply:
Created utils package